room.js

```diff
@@ -4,9 +4,17 @@ class Room{
 	constructor(currentHotel){
 		this.occupant = null; //a customer
 		this.hotel = currentHotel;
+		this.isOccupied = false;
 	}
 	addOccupant( customer ){
-
+		this.occupant = customer;
+	}
+	checkIfOccupied(){
+		if(this.isOccupied===false){
```

Comment:
You could have also checked if this.occupant is null.  This is needlessly complex for the moment.  Going to reject.